images/configMap.yaml

```diff
@@ -21,6 +21,15 @@ data:
                     "devices": ["154c", "10ed"],
                     "drivers": ["vfio-pci"]
                 }
+            },
+            {
+                "resourceName": "mlnx_sriov_rdma",
+                "isRdma": true,
```

Comment:
Is there any thought on how to add various ``rdma`` modes support based on the ``isRdma`` config?  like how do you specify the rdma mode(RoCE, IB, etc) in the config.

Reply:
How RDMA is implemented and transported depends on the fabric and the driver.
Basically you can say, from an application point of view(assuming it was written correctly and does no assumptions on the underlying fabric), that its "auto-selected".

Reply:
> What happens if a resource pool configured as `"isRdma": true` but the resource pool contains devices that do not support RDMA(insufficient selectors)? Are you ensuring that only RDMA capable devices should be included for a resource pool with `"isRdma": true`?

This approach only includes the RDMA capable devices, we may add an additional configuration's flag `"mountRdma": bool` this will be to mount RDMA resources if the device is RDMA capable, or we can mount the RDMA resource if the device is RDMA capable by default and remove `isRdma` flag from the configuration file, what do you think is the best approach?

Reply:
> How RDMA is implemented and transported depends on the fabric and the driver.
> Basically you can say, from an application point of view(assuming it was written correctly and does no assumptions on the underlying fabric), that its "auto-selected".

That is correct, but other modes of rdma may need different device specs/mounts to be exported. And we may not want to mix different types of RDMA capbale devices into same resource pool either(for different underlying fabrics. performance, latency, CPU utilization etc).